app/posts/personalised-prevention-platform/2025/03/2025-03-12-discovery-summary.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Discovery summary"
+description: A summary post detailing our discovery into personalised prevention
+date: 2025-03-12
+---
+
+## Introduction
+
+This discovery summary captures our insights into understanding the potential value of a “digital lifestyle platform” initiative as originally proposed by the Prevention and Long-Term Conditions Strategy team:
+
+- many people are at risk of developing (or already have) single or multiple health conditions
```

Comment:
These bullets are quite self-contained rather than following on from a lead-in line so I would probs keep them as bullets but start each one with a capital letter and end each one with a full stop.

Suggest changing to:

'- Many people are at risk of developing (or already have) single or multiple health conditions.
- Only a small percentage of these people are reached and engaged by healthcare services that can support them to tackle those conditions.'